examples/postgresql-kerberos/README.md

```diff
@@ -0,0 +1,78 @@
+# PostgreSQL Example
```

Comment:
I don't think we need to show this in an example - adding a directory service resource will take a looooooooooooooooooooong time to test/validate. For this change we can skip the example usage - @antonbabenko what are your thoughts

Reply:
I agree, we don't need to add such a long example. The feature itself adds just 2 new arguments to the module.

Reply:
kerberos example removed from PR.